app/controllers/api/projects.php

```diff
@@ -747,9 +755,12 @@
             throw new Exception('Project not found', 404);
         }
 
-        $webhook = $project->find('$id', $webhookId, 'webhooks');
+        $webhook = $dbForConsole->findOne('webhooks', [
+            new Query('_uid', Query::TYPE_EQUAL, [$webhookId]),
+            new Query('projectId', Query::TYPE_EQUAL, [$project->getId()])
+        ]);
 
-        if (empty($webhook) || !$webhook instanceof Document) {
+        if ($webhook == false || $webhook->isEmpty()) {
```

Comment:
I think we should use strict comparisons wherever possible
```suggestion
        if ($webhook === false || $webhook->isEmpty()) {
```